### PR TITLE
Quieting warnings

### PR DESCRIPTION
**Pull request by danielpunkass**
Mike - I'm a little confused by the ObjectiveFlickr submodule reference change included in the f1fe9a7 commit. I didn't intend to change anything there so can you selectively ignore that assume you take this pull request?

### CODE REVIEW

File: IMBiPhotoParser.m [hunk 1]
```diff
@@ -1031,7 +1031,6 @@ - (void) populateEventsNode:(IMBNode*)inNode withEvents:(NSArray*)inEvents image
                 }
                 else
                 {
-                    #warning @Jörg This is a preliminary "fix" for the Events bug with iPhoto 9.4
```
Comment (mikeabdullah) [inline IMBiPhotoParser.m:1034]: @jjac, what's going on with this at present?

Comment (jjac) [inline IMBiPhotoParser.m:1034]: Fixed for good with commit bcec4f876361cd7c5c1468bd249a409bfac8e2e4 in iMedia 3.0.

Not sure whether commit is applicable to 2.5 without adaption (e.g. parserIdentifier not present in 2.5)

Comment (mikeabdullah) [inline IMBiPhotoParser.m:1034]: Aha, a bit more hunting shows it got left in by mistake from my merge conflict resolution.

**Comment by mikeabdullah** [comment]
I can leave out the ObjectiveFlickr change indeed, just means I have to cherry-pick the changes instead of merging.

**Comment by mikeabdullah** [comment]
Effectively merged by cherry-picking.

**Pull request closed without merging**